wiki/Rules/Explicit_Content/en.md

```diff
@@ -0,0 +1,30 @@
+# Explicit Content
+
+When creating a beatmap, mappers are able to declare whether the content of their beatmap is 'explicit' by using a check box in the language/genre popup on the website.
+
+This feature is relatively new, but has a few expectations and rules governing its use.
+
+## What counts as explicit content?
+
+Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter or *heavy* use of explicit language.
+
+**It does not apply to visual elements of a beatmap - those MUST adhere to the [Visual Content Considerations](Rules/Visual_Content_Considerations).**
+
+In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as storyboards or bespoke (ie: specifically created) video content.
+
+Generally speaking, with the advent of the new explicit content marker, most forms of music are now acceptable to use in beatmaps with very few exceptions, so long as they are marked appropriately.
+
+## What counts as heavy use of explicit language?
+
+Mild (or single) use of any common profanity is not enough to qualify a beatmap for the explicit marker. Beatmaps with this kind of language can be safely left without being tagged as explicit - so long as the language is what one would reasonably expect from something classified as requiring "parental guidance" (PG-13). If any arguments break out over whether this is relevant for a given track, assume that it isn't.
+
+Profanity use must be sustained, significant, heavily inflammatory or highly repetitive in order to count as explicit.
+
+In addition, vivid and sustained discussion of controversial topics or anything that a reasonable person would consider as "heavy" should be considered as explicit. Some (but not all) examples are:
+
+- Imagery, impact or consequences of suicide
+- Heavily implied violence 
+- Depiction/discussion of violent consequences in gory detail
+- Extremely overt sexual references or allusions to sexual acts
+
+A good way of discerning if something should be tagged as explicit or not is if any major audio streaming services have it listed as such on their own platforms. Spotify is an excellent place to start.
```

Comment:
```suggestion
A good way of discerning if something should be tagged as explicit or not is if any major audio streaming services have it listed as such on their own platforms. [Spotify](https://www.spotify.com "Spotify") is an excellent place to start.
```
because why not